modules/nifti/src/niftireader.cpp

```diff
@@ -209,6 +211,9 @@ std::shared_ptr<NiftiReader::VolumeSequence> NiftiReader::readData(const std::st
     }
 
     /*
+     We currently have no good way of considering cal_min and cal_max.
```

Comment:
why can we not just use dataRange for this?

Reply:
I managed to figure out how it could be done.